util.go

```diff
@@ -37,3 +40,44 @@ func prettyPrint(data interface{}) {
 	dbg, _ := json.MarshalIndent(data, "", "  ")
 	fmt.Println(string(dbg))
 }
+
+// GetDefaultRelease attempts to guess a default release.
+func GetDefaultRelease() string {
+	if release := os.Getenv("SENTRY_RELEASE"); release == "" {
+		// Attempt to get the last commit hash with git.
+		var stdout bytes.Buffer
+		cmd := exec.Command("git", "rev-parse", "HEAD")
+		cmd.Stdout = &stdout
+		err := cmd.Run()
+		if err != nil {
+			Logger.Println("Failed attempt to run git rev-parse.")
+		} else {
+			shastr := strings.TrimSpace(stdout.String())
+			if len(shastr) == 40 { // sha1 hash length
+				return shastr
+			}
+		}
+
+		// Look for environment variables known to hold versioning info.
+		if release == "" {
+			envVars := []string{
+				"HEROKU_SLUG_COMMIT",
+				"SOURCE_VERSION",
+				"CODEBUILD_RESOLVED_SOURCE_VERSION",
+				"CIRCLE_SHA1",
+				"GAE_DEPLOYMENT_ID",
+				"GITHUB_SHA",             // GitHub Actions - https://help.github.com/en/actions
+				"COMMIT_REF",             // Netlify - https://docs.netlify.com/
+				"VERCEL_GIT_COMMIT_SHA",  // Vercel - https://vercel.com/
+				"ZEIT_GITHUB_COMMIT_SHA", // Zeit (now known as Vercel)
+				"ZEIT_GITLAB_COMMIT_SHA",
+				"ZEIT_BITBUCKET_COMMIT_SHA"}
+			for _, env := range envVars {
+				if envVal := os.Getenv(env); envVal != "" {
+					return envVal // Stop at first non-empty variable.
+				}
+			}
```

Comment:
I think it might be a better end result to check env vars first, and only as a last resort look at VCS/git, because env vars might give a more informative version name than `git`.

@AramisHM, what do you think?

If we pursue that route, this whole function could become shorter too, along the lines of:

```go
func defaultRelease() string {
	s := []string{"SENTRY_RELEASE", "GITHUB_SHA", ...}
	for _, e := range s {
		if v := os.Getenv(e); v != "" {
			return v
		}
	}
	// ... no env var had a suitable release name, try git...
	// ...

	// not able to find a release name at all
        return ""
}